sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -94,4 +94,20 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M4</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
+        <configuration>
+          <rerunFailingTestsCount>3</rerunFailingTestsCount>
+          <statelessTestsetReporter implementation="org.apache.maven.plugin.surefire.extensions.junit5.JUnit5Xml30StatelessReporter">
+            <usePhrasedTestCaseMethodName>true</usePhrasedTestCaseMethodName>
+          </statelessTestsetReporter>
```

Comment:
Does this print out the running test?

Reply:
In surefire report, the parameterized display name looks like below:

Before this change:
```
 <testcase name="testDisplay{String}[1]" classname="com.example.MyTest" time="0.029"/>
  <testcase name="testDisplay{String}[2]" classname="com.example.MyTest" time="0.002"/>
```

After this change:
```
  <testcase name="testDisplay(String) with val0" classname="com.example.MyTest" time="0.029"/>
  <testcase name="testDisplay(String) with val1" classname="com.example.MyTest" time="0.002"/>
```

Check the Test tab for new display:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=360455&view=ms.vss-test-web.build-test-results-tab

Reply:
Yeah, I saw the comment on github issue last night and it only worked for reporting and wouldn't print the tests that are running on the console. @alzimmermsft has a PR that will print all running test names.